dates.py

```diff
@@ -0,0 +1,16 @@
+RUN_DATE = "RUN_FOR_DATE"
+import datetime
+from os import environ
+
+
+def get_run_dates():
+    if RUN_DATE not in environ:
+        raise RuntimeError("Expected variable {} in environment".format(RUN_DATE))
+    today = datetime.date.fromisoformat(environ[RUN_DATE])
+
+    start_time = datetime.datetime.combine(today, datetime.time.min)
+    end_time = datetime.datetime.combine(
+        start_time + datetime.timedelta(days=1), datetime.time.min
+    )
+
+    return (start_time, end_time)
```

Comment:
Does python have something similar to Ruby's `Struct`? If so, this Pair could use a name?

Reply:
We could define a `dataclass`; it's a real class, but the syntax is short; something like:
```python
@dataclass
class TimeRange:
	start_time: datetime.datetime
	end_time: datetime.datetime
```